### ISSUE TITLE
Add option to configure a default connection in getConnection

### ISSUE THREAD

**Issue by stevleibelt** [bazzline/php_propel_behavior_entity_instantiator#3]
When a class is generated, the method "getConnection" looks like the following.

```php
    /**
     * @param null|string $name - The data source name that is used to look up the DSN from the runtime configuration file.
     * @param string $mode The connection mode (this applies to replication systems).
     * @return PDO
     */
    public function getConnection($name = null, $mode = Propel::CONNECTION_WRITE)
    {
        return Propel::getConnection($name, $mode);
    }
```

It can happen that you are using multiple databases resulting in multiple generated classes.
To still be able to get the right connection without passing around the name, it would be good to configure the default name. If the default name is configured, the generated method could look like that.

```php
    /**
     * @param null|string $name - The data source name that is used to look up the DSN from the runtime configuration file.
     * @param string $mode The connection mode (this applies to replication systems).
     * @return PDO
     */
    public function getConnection($name = null, $mode = Propel::CONNECTION_WRITE)
    {
        if (is_null($name)) {
            $name = 'my_configured_default_value';
        }

        return Propel::getConnection($name, $mode);
    }
```

The same could be done for the value "$mode".

**Comment by stevleibelt** [comment]
An other option would be to use the configured database name within the schema.xml as default.

```php
$this->getDatabase()->getName().
```

**Comment by stevleibelt** [comment]
Started working on this in branch [issue_2](https://github.com/bazzline/php_propel_behavior_entity_instantiator/tree/issue_2).

**Comment by stevleibelt** [comment]
Fixed in [release 1.1.0](https://github.com/bazzline/php_propel_behavior_entity_instantiator/tree/1.1.0) - This is for you @alex :-).
